common/culture/traditions/00_regional_traditions.txt

```diff
@@ -62,16 +62,21 @@ tradition_byzantine_succession = {
 	ai_will_do = { value = 100 }
 }
 
-# Warcraft
 tradition_lords_of_the_elephant = {
 	category = regional
 
+	# Warcraft
 	desc = {
 		first_valid = {
 			triggered_desc = {
 				trigger = {
 					has_innovation = innovation_elephantry
-					this = culture:draenei
+					OR = {
+						this = culture:draenei
+						any_parent_culture_or_above = {
+							this = culture:draenei
+						}
+					}
```

Comment:
`any_parent_culture_or_above` is redundant here because 'draenei' is the first and only culture within 'heritage_argussean'. Is your intent to extend this trigger condition to 'draenei' and its potential derivative cultures? I was also looking for a trigger like this in the vanilla files but did not find anything. I thought about creating a trigger for this purpose but was not sure where to define it.